README.md

```diff
@@ -250,13 +249,22 @@ skip-magic-trailing-comma = false
 line-ending = "auto"
 ```
 
-Some configuration options can be provided via the command-line, such as those related to
-rule enablement and disablement, file discovery, and logging level:
+Note that, in a `pyproject.toml`, each section header should be prefixed with `tool.ruff`. For
+example, `[lint]` should be replaced with `[tool.ruff.lint]`.
+
+Some configuration options can be provided via dedicated command-line arguments, such as those
+related to rule enablement and disablement, file discovery, and logging level:
 
 ```shell
 ruff check path/to/code/ --select F401 --select F403 --quiet
 ```
 
+The remaining configuration options can be provided through a catch-all `--config` argument:
+
+```shell
+ruff check path/to/code/ --config "lint.per-file-ignores = {'some_file.py' = ['F841']}"
```

Comment:
I don't love `path/to/code/` here it seems like a bad example as a corollary for file discovery options.

I know it's copied from above but maybe a good time to improve